spec/models/conference_spec.rb

```diff
@@ -1656,4 +1656,32 @@
       expect{ conference.save }.to change{ EventSchedule.count }.from(2).to(1)
     end
   end
+
+  describe 'revision count' do
+
+    let(:track) { create(:track, program: subject.program) }
+    let(:event) { create(:event, program: subject.program, track: track) }
+    let(:venue) { create(:venue, conference: subject) }
+    let(:room)  { create(:room, venue: venue) }
+
+    context 'should increment' do
+
+      it 'for change in conference' do
+        subject.title = 'changed'
+        expect{ subject.save }.to change { subject.revision }.by(1)
+      end
+      it 'for change in event' do
```

Comment:
Please put a blank line between `it` blocks :)
(I wonder why we don't have a RuboCop rule for this...)